### DISCUSSION TITLE
Why the relationship be called `Edges`?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [antvis/G6#4220] [Q&A] [answered]
>● Nodes ： 用户关心的实体节点
● Edges：实体间的关联关系
● Property：节点和边的次要信息或描述特征

Why the relationship be called `Edges`?

**Comment by curious-broccoli** [comment, marked as answer]
It's simply a common name for the link between nodes, see https://en.wikipedia.org/wiki/Graph_theory
